tests/bsim/bluetooth/host/adv/chain/src/main.c

```diff
@@ -94,25 +98,29 @@ static void scan_recv(const struct bt_le_scan_recv_info *info,
 
 	data_len = buf->len;
 	if (data_len != DATA_LEN) {
+		printk("Unexpected length\n");
 		return;
 	}
 
 	(void)memset(name, 0, sizeof(name));
 	bt_data_parse(buf, data_cb, name);
 
 	if (strcmp(name, CONFIG_BT_DEVICE_NAME)) {
+		printk("Wrong name\n");
 		return;
 	}
 
 	for (uint8_t i = 0; i < sid_count; i++) {
 		if (sid[i] == info->sid) {
+			printk("Wrong SID\n");
 			return;
 		}
 	}
 
 	sid[sid_count++] = info->sid;
 
 	if (sid_count < CONFIG_BT_EXT_ADV_MAX_ADV_SET) {
+		printk("Wrong number of advertising sets\n");
```

Comment:
Ditto

Reply:
Did not fail in the original test. Updated text to reflect this better